CMakeLists.txt

```diff
@@ -111,10 +115,29 @@ if(MSVC)
     endif(BOOST_ALL_DYN_LINK)
 endif(MSVC)
 
-include(VolkBoost)
+# Check if we have std::filesystem
+if(POLICY CMP0057)
+    cmake_policy(SET CMP0057 NEW)
+endif()
+
+if(CMAKE_VERSION VERSION_LESS 3.8)
```

Comment:
One more good reason to require CMake 3.8.

Reply:
ditto CMake version fixup